package.json

```diff
@@ -645,6 +673,41 @@
                     "command": "appService.RemoveCosmosDBConnection",
                     "when": "view == azureAppService && viewItem == cosmosDBConnection",
                     "group": "2@1"
+                },
+                {
+                    "command": "appService.OpenInPortal",
+                    "when": "view == azureAppService && viewItem =~ /^deployments(C|Unc)onnected$/",
+                    "group": "1@1"
+                },
+                {
+                    "command": "appService.DisconnectRepo",
+                    "when": "view == azureAppService && viewItem == deploymentsConnected",
+                    "group": "1@2"
+                },
+                {
+                    "command": "appService.Refresh",
+                    "when": "view == azureAppService && viewItem =~ /^deployments(C|Unc)onnected$/",
+                    "group": "2@1"
+                },
+                {
+                    "command": "appService.Redeploy",
+                    "when": "view == azureAppService && viewItem == deployment",
+                    "group": "1@1"
+                },
+                {
+                    "command": "appService.OpenInPortal",
+                    "when": "view == azureAppService && viewItem == deployment",
+                    "group": "1@2"
+                },
+                {
+                    "command": "appService.Refresh",
+                    "when": "view == azureAppService && viewItem == deployment",
+                    "group": "2@1"
+                },
+                {
+                    "command": "appService.ShowDeploymentLogs",
+                    "when": "view == azureAppService && viewItem == deployment",
+                    "group": "inline"
                 }
             ],
             "explorer/context": [
```

Comment:
You should bump the minimum requirement of 'vscode-azureappervice' package

Reply:
Isn't v0.25.0 the most recent release?

Reply:
Yes but your features will rely on 0.25.1 so the minimum should be 0.25.1, not 0.25.0. If someone installs 0.25.0 in their repo it won't work.